pkg/platform/src/components/secret.ts

```diff
@@ -98,7 +98,7 @@ export class Secret extends Component implements Link.Linkable {
     this._name = name;
     this._placeholder = placeholder;
     const value = process.env["SST_SECRET_" + this._name] ?? this._placeholder;
-    if (!value) {
+    if (typeof value === "undefined") {
```

Comment:
Makes sense to me fwiw (not a maintainer), I might flip the logic but that's a nit

```suggestion
    if (typeof value !== "string") {
```